src/frontend/entry-button.c

```diff
@@ -216,6 +215,9 @@ static void brisk_menu_entry_button_init(BriskMenuEntryButton *self)
         gtk_misc_set_alignment(GTK_MISC(self->label), 0.0, 0.5);
         G_GNUC_END_IGNORE_DEPRECATIONS
 
+        menu = gtk_menu_new();
```

Comment:
Can probably just directly set `self->menu = gtk_menu_new()` here, no need for the second `menu` pointer really